### PR TITLE
Update the readme

### PR DESCRIPTION
**Pull request by rhnoble**
This change is [Reviewable](https://reviewable.io/reviews/plangrid/android-skeleton/2)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # android-skeleton
 A skeleton app to make sure we've got everything up to date
+
```
Comment (rhnoble) [inline README.md]: Some comment

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+Make sure your local environment can run this app as is, so we don't spend half the interview downloading sdks.
```
Comment (rhnoble) [inline README.md]: Some other comment

**Review by rhnoble — COMMENTED** [review]
Some stuff

**Comment by rhnoble** [comment]
*[README.md, line 4 at r1](https://reviewable.io:443/reviews/plangrid/android-skeleton/2#-L7_XMAC1NcF_3qIdSB-:-L7_XMAC1NcF_3qIdSB0:bukbcp4) ([raw file](https://github.com/plangrid/android-skeleton/blob/f979131b69fc2f4bac8e59bafd4fa7a71f11de33/README.md#L4)):*
> ```Markdown
> A skeleton app to make sure we've got everything up to date
> 
> Make sure your local environment can run this app as is, so we don't spend half the interview downloading sdks.
> ```

Some other comment

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/plangrid/android-skeleton/2)*

**Comment by mjanes** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/plangrid/android-skeleton/2#-:-L7_jABX8cXToACKao2Y:bnfp4nl)*

**Comment by mjanes** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/plangrid/android-skeleton/2)*

**Comment by MisterRager** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/plangrid/android-skeleton/2#-:-L7_lPMeQBnMbC84uBX4:bnfp4nl)*

**Comment by robjailall** [comment]
I don’t need to be getting these. Can you remove me from reviewable

**Comment by rhnoble** [comment]
@robjailall Sorry for the spam. Reviewable doesn't store user info, emails are coming via GitHub notifications. As far as I can tell there's nothing on our end that's connecting you to this project. I'm not sure why you're getting these, but since this is a public project I think you'll have to unsubscribe from it on your end via GitHub.

If that's not working for you drop me an email (harry@plangrid.com) so we can take the conversation offline.

**Comment by robjailall** [comment]
ah, its public. thats why . ill remove myself

**Comment by rhnoble** [comment]
*[README.md, line 4 at r1](https://beta.reviewable.io/reviews/plangrid/android-skeleton/2#-L7kG17p4GwYrMnTQ0Sf:-L7kG17p4GwYrMnTQ0Sg:btd2n5f) ([raw file](https://github.com/plangrid/android-skeleton/blob/f979131b69fc2f4bac8e59bafd4fa7a71f11de33/README.md#L4)):*
> ```Markdown
> A skeleton app to make sure we've got everything up to date
> 
> Make sure your local environment can run this app as is, so we don't spend half the interview downloading sdks.
> ```

One more new comment

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/plangrid/android-skeleton/2)*

**Comment by mjanes** [comment]
:lgtm:

**Pull request closed without merging**